_projects/2023/100505771/100505771.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "Mexico's Export Dynamics from 1996 to 2020 by Sector"
+description: |
+  Replication and alternative version of the over time graph sector exports of Mexico by the Growth Lab at Harvard University.
+author: Valeria Contreras
+date: 2023-12-12
```

Comment:
Please use ```date: "`r Sys.Date()`"``` instead. The date will be automatically generated when the document is knitted.